src/app/views/dashboard/games/manage/game/packages/release/edit/edit.ts

```diff
@@ -77,8 +77,7 @@ export default class RouteDashGamesManageGamePackageReleaseEdit extends BaseRout
 
 		// If the game was entered into a jam that locks its builds.
 		this.areBuildsLockedByJam = $payload.areBuildsLockedByJam || false;
-		this.areWebBuildsLockedBySellable =
-			$payload.package.is_in_paid_sellable || $payload.package.has_sales || false;
+		this.areWebBuildsLockedBySellable = $payload.package.is_in_paid_sellable || false;
```

Comment:
Actual fix is here

Reply:
And we removed the "has_sales" because we want to allow "pay what you want" and if they switched it away from "paid" sellable at some point?